admin-dev/themes/default/template/header.tpl

```diff
@@ -301,8 +301,9 @@
               {if isset($debug_mode) && $debug_mode == true}
                 <span class="shop-state" id="debug-mode">
                   <i class="material-icons">bug_report</i>
-                  <span class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
-                    title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in debug mode.'}</strong></p><p class='text-left'>{l s='All the PHP errors and messages are displayed. When you no longer need it, [1]turn off[/1] this mode.' tags=['<strong>']}</p>">{l s='Debug mode'}</span>
+                  <a class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
```

Comment:
uhh, i think that it should be accepted as is and changed globally after that...